tests/php/bootstrap.php

```diff
@@ -13,3 +13,6 @@
 
 // Load our plugin.
 require dirname( dirname( __DIR__ ) ) . '/secure-custom-fields.php';
+
+// Load the ACF_Form_Post class.
+require dirname( dirname( __DIR__ ) ) . '/includes/forms/form-post.php';
```

Comment:
Can you share your thought on adding it to the main bootstrap? My first thought without digging into anything would be to have this require within the test-form-post.php file.

Reply:
I required the file on the test one, as you mentioned. Honestly, I'm not 100% sure of the best approach for these unit tests. Let's see in the future for functions that require several files.